test/cli/test_cct.yaml

```diff
@@ -0,0 +1,166 @@
+exe: cct
+tests:
+- args: -d 8 +proj=merc +R=1
+  in: 90 45 0
+  out: |2
```

Comment:
what is the meaning of this "|2" ?

Reply:
This is an indentation indicator, which is specified here since `cct` output has leading whitespace. A great resource on the topic that I've used for this PR is https://yaml-multiline.info/